### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by rov9**
E commerce Products Page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+                    <h2 class="product-brand">brand 4</h2>
+                    <p class="product-short-des">short description</p>
+                    <span class="price">$20</span><span class="actual-price">$40</span>
+                </div>
+            </div>
+
+            <div class="product-card">
+                <div class="product-image">
+                    <span class="discount-tag">50% off</span>
+                    <img src="/product-img.jpg" class="product-thumb" alt="">
```
Comment (iamsaurav28) [inline index.html:130]: give naming to alt

File: index.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                    <h2 class="product-brand">brand 5</h2>
+                    <p class="product-short-des">short description</p>
+                    <span class="price">$20</span><span class="actual-price">$40</span>
+                </div>
+            </div>
+
+            <div class="product-card">
+                <div class="product-image">
+                    <span class="discount-tag">50% off</span>
+                    <img src="/product-img.jpg" class="product-thumb" alt="">
```
Comment (iamsaurav28) [inline index.html:143]: same as above

File: style.css [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+    height: 20px;
+    line-height: 20px;
+    overflow: hidden;
+    opacity: 0.5;
+    text-transform: capitalize;
+    margin: 5px 0;
+}
+
+.price{
+    font-weight: 900;
```
Comment (iamsaurav28) [inline style.css:234]: give units after numbers

**Review by iamsaurav28 — APPROVED** [review]

File: index.html [hunk 4]
```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Listing</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://xpressui1.netlify.app/xPressUi.css">
```
Comment (tanishq20) [inline index.html:9]: You can import the component library link directly in the main CSS.
By doing this you will not have to give links in all HTML files.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <script src="https://kit.fontawesome.com/2ea00fb38c.js" crossorigin="anonymous"></script>
+
+</head>
+<body>
+
+    <nav class="navbar">
+        <div class="nav">
+           <!-- <img src="/logo.png" class="brand-logo" alt="" > -->
```
Comment (tanishq20) [inline index.html:19]: remove comments

File: index.html [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+
+
+    <section class="product">
+        <h2 class="product-category">best selling</h2>
+        
+        <div class="product-container">
+            <div class="product-card">
+                <div class="product-image">
+                    <span class="discount-tag">50% off</span>
+                    <img src="/product-img.jpg" class="product-thumb" alt="">
```
Comment (tanishq20) [inline index.html:79]: In `img`, alt can't be empty. Give some value.
(Apply this to all)

File: style.css [hunk 7]
```diff
@@ -0,0 +1,341 @@
...
+.product{
+    position: relative;
+    overflow: hidden;
+    padding: 20px 0;
+    margin-left:250px;
+}
+
+.product-category{
+    padding: 0 10vw;
+    font-size: 30px;
```
Comment (tanishq20) [inline style.css:19]: Prefer to use `rem`

File: style.css [hunk 8]
```diff
@@ -0,0 +1,341 @@
...
+}
+
+.ecom-sidenav {
+    height:fit-content;
+    width: 250px;
+    position: fixed;
+    z-index: 1;
+    top: 0;
+    left: 0;
+    /* background-color: rgb(255, 251, 251); */
```
Comment (tanishq20) [inline style.css:32]: remove comments

File: style.css [hunk 9]
```diff
@@ -0,0 +1,341 @@
...
+
+  .sidenav-header{
+      padding: 6px 8px 6px 16px;
+  }
+  
+  .py-3{
+    padding:6px 8px 6px 16px;
+    text-decoration: none;
+    font-size: 13px;
+    color: #000000;
```
Comment (tanishq20) [inline style.css:47]: use var() for colors
(apply this to all)

File: style.css [hunk 10]
```diff
@@ -0,0 +1,341 @@
...
+
+.search{
+    width: 600px;
+    display: flex;
+}
+
+.search-box{
+    width: 80%;
+    height: 40px;
+    /* padding: 10px; */
```
Comment (tanishq20) [inline style.css:83]: remove comment
(apply this to all)

File: style.css [hunk 11]
```diff
@@ -0,0 +1,341 @@
...
+    outline: none;
+}
+
+
+.search-btn{
+    width: 20%;
+    height: 40px;
+    padding: 10px 20px;
+    border: none;
+    outline: none;
```
Comment (tanishq20) [inline style.css:99]: Avoid giving none properties.
Read more about it when you are free.

File: style.css [hunk 12]
```diff
@@ -0,0 +1,341 @@
...
+.price{
+    font-weight: 900;
+    font-size: 20px;
+}
+
+.actual-price{
+    margin-left: 20px;
+    opacity: 0.5;
+    text-decoration: line-through;
+} */
```
Comment (tanishq20) [inline style.css:341]: If not in use then please remove it.

File: .vscode/settings.json [hunk 13]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
```
Comment (tanishq20) [inline .vscode/settings.json:2]: Avoid opening two HTML files in the Live server at the same time. 
This causes a change in setting.json.

**Review by tanishq20 — COMMENTED** [review]
Suggested some changes.

**Pull request merged**